### ISSUE TITLE
Existing Design Work

### ISSUE THREAD

**Issue by melvincarvalho** [NostrGit/NostrGit#39]
Thread is a place holder for existing design work that could act as an inspiration in future

Lots of interesting design stuff here from githubnext

- https://githubnext.com/projects/workspaces/

- https://blocks.githubnext.com/

**Comment by cypherhoodlum** [comment]
Intriguing stuff. I like the part where the repo could be navigated through the documentation to help bootstrap new developers. However, I think we should avoid using GitHub-spesific tools because the plan is to migrate the repo to NostrGit when able. I think it's revolutionary enough to be able to collaborate on a project over nostr (that's the goal). I fear it could be antithetical to NostrGit to collaborate using githubnext. Anyone feels different?

**Comment by PeerRich** [comment]
renamed thread

**Comment by PeerRich** [comment]
found this today, super epic:

https://github.com/YousefED/nostr-crdt


real time with nostr


dont think we need anything realtime in nostrgit but worth sharing

**Comment by PeerRich** [comment]
https://damus.io/note1j3y8mktxluyxuvw053j0fg392n3n7drlzmen03euvkhkhyymtkvs6p0c6c


interesting 

https://github.com/hackergrrl/git-ssb-intro

**Comment by cypherhoodlum** [comment]
> dont think we need anything realtime in nostrgit but worth sharing

Yeah, things are happening incredibly fast as is :smile: Hard to keep up with the backlog of all the things you have already managed to do @PeerRich @melvincarvalho @brycedev @pierresisson ! :rocket:
